src/serverrpc.cpp

```diff
@@ -119,7 +119,7 @@ CServerRpc::CServerRpc ( CServer* pServer, CRpcServer* pRpcServer, QObject* pare
             { "city", pServer->GetServerCity() },
             { "countryId", pServer->GetServerCountry() },
             { "welcomeMessage", pServer->GetWelcomeMessage() },
-            { "directoryServer", dsName },
+            { "directoryServer", dsName }, // TODO rename to 'directory' and update doccomment above
```

Comment:
I would change right here. The API can have breaking changes.

Reply:
I'd rather do it in a separate change.  This one is explicitly _not_ meant to make functional changes.  I'll try to remember to base a new change off this with the fixes.

Reply:
See https://github.com/pljones/jamulus/tree/bugfix/rpc-remaining-directory-server-mentions
(I don't see an easy way to compare that branch to the source of this pull request.)